### ISSUE TITLE
Submitted solution for Kata "Square Every Digit" & "Caffein Script". Own solution not shown under "Solutions/My Solution"

### ISSUE THREAD

**Issue by markustheelen** [Codewars/codewars.com#691]
Hi all,

i submitted solutions for the Kata "Square Every Digit" & "Caffein Script" just recently. Although the solution is accepted the solution is not shown when going to the individual Kata and navigating to "Solutions/Show me: My Solution". It says there are no solutions there yet for this kata. If however i go to my personal profile to the area "Solutions" my solution is shown there. I tried resubmitting a solution as example for the Caffein Script kata. As effect it shows as "Refactored" in my personal profile but still not Kata itself.

Would be great if you could resolve.

Many thanks in advance

**Comment by SlyBeetle** [comment]
Same problem with [Playing with digits](https://preview.codewars.com/kata/playing-with-digits/csharp).

**Comment by SlyBeetle** [comment]
Same problem with [Build a pile of Cubes](https://www.codewars.com/kata/5592e3bd57b64d00f3000047/solutions/csharp/me/best_practice).

**Comment by SlyBeetle** [comment]
After code refactoring of [Build a pile of Cubes](https://www.codewars.com/kata/5592e3bd57b64d00f3000047/solutions/csharp/me/best_practice) both are shown now.

**Comment by SlyBeetle** [comment]
But [Build a pile of Cubes](https://www.codewars.com/kata/5592e3bd57b64d00f3000047/solutions/csharp/me/best_practice) is shown without refactoring. :(

**Comment by SlyBeetle** [comment]
I think that the updating is too slow. :(

**Comment by jhoffner** [comment]
Yeah the solution has to be compared to every other solution to see if its similar and then grouped. It can be an expensive operation. If the queue is really busy even kata with only a few solutions might get processed after the page is already loaded. You can always click the "compare to my solution" though. If that doesn't show up then its not just a timing issue.

**Comment by anter69** [comment]
I just want to note that I have the same issue, e.g. with [this kata](https://www.codewars.com/kata/write-number-in-expanded-form) (in python): I have 2 different, working solutions, but they don't show up on the solutions page.

I understand the underlying issue, but it's hard to get votes on them if they are invisible... :smiley:

**Comment by kazk** [comment]
Similar issue: #33

**Comment by KDCinfo** [comment]
Just wondering if there has been any progress on this? I've looked at issues #33, #265, and #282, and although they're all still open, this one has the most recent activity. 

I've experienced my solutions not showing up in 'My Solutions' quite a few times, but maybe half the time when I go through the refactor link (as accessed from my profile solutions), and resubmit (as-is), it will then show up. Not always though. 

My latest kata that didn't work after refactoring was: https://www.codewars.com/kata/primes-in-numbers/javascript

Thanks!
